client/src/pages/LiveTelemetry.tsx

```diff
@@ -43,38 +49,24 @@ function LiveTelemetry() {
           //Calculate speed
           const rpm = response?.mitsuba?.rx0?.motorRPM ?? 0;
           setSpeed(rpm * 60 * telemetry.WHEEL_RADIUS_MI);
-
-          // The password may needed to reset but it's actually empty so it didn't
-          if (localStorage.getItem("passwordNeedsSet") == "true") {
-            localStorage.setItem("passwordNeedsSet", "false");
-            window.location.reload();
-          }
         })
         .catch((reason) => {
-          // clear username/password if it changed for some reason
-          if (
-            reason.request.status == 403 &&
-            (!localStorage.getItem("passwordNeedsSet") ||
-              localStorage.getItem("passwordNeedsSet") == "false")
-          ) {
-            localStorage.setItem("passwordNeedsSet", "true");
-            localStorage.setItem("username", "");
-            localStorage.setItem("password", "");
+          if (reason.request.status == 403) {
             window.location.reload();
           }
```

Comment:
You still need to reset the password/username since a 403 means the current password/username is invalid

Reply:
fixed in latest commit.